vendor/directx/dxgi/dxgidebug.odin

```diff
@@ -139,3 +139,21 @@ IDebug1_VTable :: struct {
 	DisableLeakTrackingForThread:   proc "stdcall" (this: ^IDebug1),
 	IsLeakTrackingEnabledForThread: proc "stdcall" (this: ^IDebug1) -> BOOL,
 }
+
+
+GetDebugInterface: proc "stdcall" (riid: ^IID, ppDebug: ^rawptr) -> HRESULT
+// Call this to start using GetDebugInterface
+debug_interface_init :: proc() -> bool {
```

Comment:
Should this follow the same naming conventions as D3D? e.g. `InitDebugInterface`

Reply:
No, it's just a helper function

Reply:
But why not?

Reply:
In GL package, helper functions do not follow GL convention, for example

Reply:
Okay, I'll change it then